parserator_web/static/css/custom.css

```diff
@@ -1 +1,13 @@
 /* Add custom style overrides here. */
+
+.result {
+    display: none;
```

Comment:
Definitely understand this concern. I set initial styles in the CSS in the hopes of avoiding a Flash of Unstyled Content, since some of those results elements have text initially. Did a quick test to show that that does indeed happen if `.hide()` is called on `document.ready`:

![fouc](https://user-images.githubusercontent.com/11164294/95932556-29a90700-0d9a-11eb-9cca-a1dac15fe0b1.gif)

I'm open to suggestions on getting all that logic into the JS without causing this issue!

Reply:
Tricky! Looks like total consolidation isn't a great option, then.

Bootstrap provides a `d-none` class but unfortunately it's marked as `!important`, so `hide()` / `show()` doesn't have the desired effect. 

A couple of ideas:

- Separating out `display: none` into its own class, and giving that class to initially hidden elements, could address my reservation about combining styles that won't always belong together under one class name.
- In the past, I've used `d-none`, and [adding and removing it](https://github.com/datamade/bga-payroll/blob/78aea94ef7c71db923d41a66684c323ee022682d/bga_database/static/js/employer.js#L137-L135) instead of using `hide()` / `show()`.

No required changes here, just wanted to drop a note in this thread. 🙃